ios/FluentUI/Drawer/DrawerPresentationController.swift

```diff
@@ -315,8 +318,12 @@ class DrawerPresentationController: UIPresentationController {
         var contentFrame = bounds.inset(by: marginsForContentView())
 
         var contentSize = presentedViewController.preferredContentSize
+        let landscapeMode = contentFrame.width > contentFrame.height
+
         if presentationDirection.isVertical {
-            if contentSize.width == 0 || traitCollection.horizontalSizeClass == .compact {
+            if contentSize.width == 0 ||
+                (traitCollection.userInterfaceIdiom == .phone && landscapeMode && shouldUseWindowFullWidthInLandscape) ||
+                (traitCollection.horizontalSizeClass == .compact && !landscapeMode) {
```

Comment:
what about iPad in compact view, while the iPad is in landscape mode?

Reply:
even in iPad compact mode in device landscape orientation, it would look more like iPhone portrait. wouldn't it be?

![Simulator Screen Shot - iPad Pro (12 9-inch) (3rd generation) - 2020-06-02 at 20 12 24](https://user-images.githubusercontent.com/20715435/83591760-86073a80-a50d-11ea-84af-c57fc3e7ee21.png)

![Simulator Screen Shot - iPad Pro (12 9-inch) (3rd generation) - 2020-06-02 at 20 12 28](https://user-images.githubusercontent.com/20715435/83591764-89022b00-a50d-11ea-9499-c1e692a2aec6.png)

![Simulator Screen Shot - iPad Pro (12 9-inch) (3rd generation) - 2020-06-02 at 20 12 34](https://user-images.githubusercontent.com/20715435/83591766-8bfd1b80-a50d-11ea-9be8-3ebe6dc9e2c3.png)